mysql-wordpress-pd/README.md

```diff
@@ -31,10 +38,10 @@ your editor added one.
 
 ```shell
 tr --delete '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/examples/master/mysql-wordpress-pd/local-volumes.yaml
```

Comment:
I'll cc @cody-clark we should probably unify the way tutorial materials are downloaded later when we start editing these tutorials.